jetty-core/jetty-util/src/test/java/org/eclipse/jetty/util/resource/ResourceTest.java

```diff
@@ -397,6 +398,7 @@ public void testEqualsWindowsAltUriSyntax() throws Exception
 
     @Test
     @EnabledOnOs(OS.WINDOWS)
+    @Disabled("This will create different Resource objects, not sure if this is still results in a problem")
```

Comment:
How does this test fail ?  Can you provide toStrings of a and b?

Reply:
There was a time when the case of the drive mattered.
And the Resource returned would be equal to each other.
But this doesn't look like a modern concern.
Incidentally, this test doesn't work exactly in tag `jetty-12.0.0` too.  Well sort of, in version 12.0.0, both Resource objects are null, as the files do not exist. So it's been a dead test for a while.